docs/_docs/includes.md

```diff
@@ -97,15 +103,18 @@ This include contains 5 parameters:
 You could also use `if` tags to safeguard scenarios where either the parameter is optional or where the author doesn't include the parameter:
 
 ```liquid
-{% raw %}<figure>{% if {{include.url}} %}<a href="{{include.url}}">
-{% endif %}<img src="{{include.file}}" {% if {{include.max-width}}
-%} style="max-width: {{include.max-width}}" {% endif %}
-alt="{{include.alt}}" />{% if {{include.url}} %}</a>{% endif %}
-{% if {{include.caption}} %}<figcaption>{{include.caption}}
-</figcaption>{% endif %}</figure>{% endraw %}
+{% raw %}<figure>
+   {% if {{ include.url }} %}<a href="{{ include.url }}">{% endif %}
+   <img src="{{ include.file }}" {% if {{ include.max-width }} %}
+   style="max-width: {{ include.max-width }}" {% endif %}
+   alt="{{ include.alt }}" />{% if {{ include.url }} %}</a>{% endif %}
+   {% if {{ include.caption }} %}
+   <figcaption>{{ include.caption }}</figcaption>
+   {% endif %}
+</figure>{% endraw %}
```

Comment:
removing the space around variables may "work" within `{% if }` blocks but Liquid should be issuing a warning it's wrong nonetheless..
I'm not in favor of Jekyll's official documentation stating this.

/cc @pathawks  your opinion?